### DISCUSSION TITLE
Error Loading App

### DISCUSSION THREAD

**Discussion by syeillah** [Budibase/budibase#4593] [Help]
Hi, i just released my app to production, but when i try to open (via chrome) there was an error message like this. 

![Screenshot_2022-02-21-10-29-36-511_com android chrome](https://user-images.githubusercontent.com/99949174/154885015-b09cb83d-15b0-40dc-aebb-fa4b5d0cdde6.jpg)

May i know why?

**Comment by mjashanks** [comment]
Not sure - does this happen with all of your apps? Does republishing work?

If you could open up your browser's dev tools, then the network tab, you should see a request to `budibase-client.js?....` what is the response for you? I.e. both in the `Headers` and `Preview` tab?

![image](https://user-images.githubusercontent.com/3524181/154979367-4bc71182-bb24-4890-8453-09f21f528c7e.png)
